src/main.rs

```diff
@@ -80,6 +80,9 @@ pub struct InstallOpts {
     /// Relative or full path for the export file that will be generated. If no path is provided, the file will be generated under home directory (https://docs.rs/dirs/latest/dirs/fn.home_dir.html).
     #[arg(short = 'f', long)]
     pub export_file: Option<PathBuf>,
+    /// The path to which the espup configuration file will be written to.
+    #[arg(long = "config")]
```

Comment:
```suggestion
    #[arg(short = 'p', long, required = false)]
```
I would go for `--config-path` as long argument name and also add a short version of it.

Reply:
`required = false` shouldn't be necessary, as this is implicitly the case for `Option<>` parameters.

Should I add id nonetheless?